plugins/search-react/src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -106,6 +106,7 @@ export const CheckboxFilter = (props: SearchFilterComponentProps) => {
       const items = checked ? [...rest, value] : rest;
       return items.length ? { ...others, [name]: items } : others;
     });
+    setPageCursor(undefined);
```

Comment:
@backstage/techdocs-core do these changes look alright?

Reply:
The same as commented in the `SearchFilter` autocomplete 🙂